src/bugsnag_app.erl

```diff
@@ -35,7 +35,11 @@ start() ->
                 _ ->
                     ok
             end,
-            bugsnag_sup:start_link({ApiKey, ReleaseState});
+            Opts = #{
+                api_key => list_to_binary(ApiKey),
+                release_stage => list_to_existing_atom(ReleaseState)
```

Comment:
`list_to_existing_atom/1` only works with atom names that already exist in the Erlang VM. I think this will be a problem. 

Maybe something like this?
```suggestion
            ReleaseStageAtom = case ReleaseState of
                undefined -> development;
                _ -> list_to_atom(ReleaseState)
            end,
            Opts = #{
                api_key => list_to_binary(ApiKey),
                release_stage => ReleaseStageAtom
```

Reply:
Good point, let's keep it as a binary maybe, to keep configuration flexible 👌🏽